pgrx-pg-config/src/lib.rs

```diff
@@ -495,6 +495,7 @@ impl<'a> PgConfigSelector<'a> {
 pub enum PgrxHomeError {
     #[error("You don't seem to have a home directory")]
     NoHomeDirectory,
+    // Do not print PathBuf due to potential security concerns
```

Comment:
perhaps "allow caller to decide whether it is safe to enumerate paths"?

Reply:
Done, my turn:  :bicyclist: :house: :paintbrush: